File: Data Structures & Algorithms/remove-element/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} val
     * @return {number}
     */
    removeElement(nums, val) {

            let i =0, 
            n = nums.length - 1;
            while(i<=n){
                if(nums[i] == val){
                    nums[i] = nums[n--];
                }else{
                    i++;
                }
            }
            return n+1;    
           

    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-0.cpp
#include<bits/stdc++.h>
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        map<int , int >mpp;

        for(int i = 0; i<nums.size(); i++){
            int num = nums[i];
            int neededMore = target - num;

            if(mpp.find(neededMore) != mpp.end()){
                return {mpp[neededMore] , i};
            }
            mpp[num]=i;
        }
        return {-1, -1};
    }
};
